lib/Style/Normalizer/Border.php

```diff
@@ -25,8 +25,8 @@ public function normalize($ruleValue, string $ruleName = ''): array
 		}
 		$matches = [];
 		preg_match('/([0-9]+)([a-z]+)\s+(solid|dashed|dotted|none)\s+(.+)?/ui', $ruleValue, $matches);
-		$originalSize = $matches[1];
-		$originalUnit = $matches[2];
+		$originalSize = $matches[1] = 0;
```

Comment:
Are you sure this is the correct code?
Maybe something like this:
```suggestion
		$originalSize = $matches[1] ?? 0;
```